### DISCUSSION TITLE
GPG Verification

### DISCUSSION THREAD

**Discussion by chuckn246** [kovidgoyal/kitty#5942] [General]
The releases are GPG signed but I couldn't find the author's GPG key to import on GitHub or the website.

I did eventually find it on the Ubuntu keyserver by searching the key from the output:
```
$ gpg --verify kitty-0.26.5.dmg.sig kitty-0.26.5.dmg                 [0]
gpg: Signature made Mon Nov  7 05:47:29 2022 EST
gpg:                using RSA key 3CE1780F78DD88DF45194FD706BC317B515ACE7C
gpg: Can't check signature: No public key

$ gpg --keyserver keyserver.ubuntu.com --search-key 3CE1780F78DD88DF45194FD706BC317B515ACE7C
```

Is the author's GPG key published somewhere and I just overlooked it?

It would be nice to add verification steps and a link to the key to the install instructions for those who wish to do so.

**Comment by page-down** [comment]
> ... I couldn't find the author's GPG key to import on GitHub

```text
https://github.com/kovidgoyal.gpg
```

> or the website.

```text
https://calibre-ebook.com/signatures/kovid.gpg
```

The link is already given on the GitHub Releases page.
https://github.com/kovidgoyal/kitty/releases

**Comment by chuckn246** [comment]
> The link is already given on the GitHub Releases page.
https://github.com/kovidgoyal/kitty/releases

Ahh - right in front of my face, thanks!

The gpg key is under the calibre-web section of the site, I was looking at the kitty pages - but the link in the releases would've pointed me in the right direction.

(I still think that adding instructions and a link under Manual Installation would be nice)
